seatunnel-datasource/seatunnel-datasource-plugins/datasource-console/src/main/java/org/apache/seatunnel/datasource/plugin/console/ConsoleDataSourceConfig.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.datasource.plugin.console;
+
+import org.apache.seatunnel.api.configuration.Option;
+import org.apache.seatunnel.api.configuration.Options;
+import org.apache.seatunnel.api.configuration.util.OptionRule;
+import org.apache.seatunnel.datasource.plugin.api.DataSourcePluginInfo;
+import org.apache.seatunnel.datasource.plugin.api.DatasourcePluginTypeEnum;
+
+public class ConsoleDataSourceConfig {
+
+    public static final String PLUGIN_NAME = "Console";
+
+    public static final DataSourcePluginInfo CONSOLE_DATASOURCE_PLUGIN_INFO =
+            DataSourcePluginInfo.builder()
+                    .name(PLUGIN_NAME)
+                    .icon(PLUGIN_NAME)
+                    .version("1.0.0")
+                    .type(DatasourcePluginTypeEnum.FAKE_CONNECTION.getCode())
+                    .build();
+
+    public static final Option<String> URL =
+            Options.key("url")
+                    .stringType()
+                    .defaultValue("dummy://URL")
+                    .withDescription(
+                            "This is required to meet the design constraints, however, it won't be utilized during execution.");
```

Comment:
How about use this option rule? https://github.com/apache/seatunnel/pull/7350/files

Reply:
I initially considered making this an optional option or even eliminating it entirely.
However, due to the database design requirements, some form of value is necessary.
This information will be stored in the "datasource_config" field of the "t_st_datasource" table, which expects a non-null value.

Table Reference:
```
CREATE TABLE `t_st_datasource`  (
  `id` bigint(20) NOT NULL,
  `datasource_name` varchar(63) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
  `plugin_name` varchar(63) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
  `plugin_version` varchar(63) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NULL DEFAULT '1.0.0',
  `datasource_config` varchar(1023) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
  `description` varchar(63) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NULL DEFAULT NULL,
  `create_user_id` int(11) NOT NULL,
  `update_user_id` int(11) NOT NULL,
  `create_time` timestamp(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
  `update_time` timestamp(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3) ON UPDATE CURRENT_TIMESTAMP(3),
  PRIMARY KEY (`id`) USING BTREE,
  UNIQUE INDEX `t_st_datasource_datasource_name_uindex`(`datasource_name`) USING BTREE
) ENGINE = InnoDB CHARACTER SET = utf8mb4 COLLATE = utf8mb4_bin ROW_FORMAT = Dynamic;
```

Reply:
In fact, console has two option named `LOG_PRINT_DATA` and `LOG_PRINT_DELAY` in https://github.com/apache/seatunnel/blob/862e2055cec6a4428ec4cf67b472ba41ca364d05/seatunnel-connectors-v2/connector-console/src/main/java/org/apache/seatunnel/connectors/seatunnel/console/sink/ConsoleSinkFactory.java#L34

So I think we can replace useless `URL` option by this two.

Reply:
From my understanding, we are dealing with two components here: the datasource and the connector. 
The options LOG_PRINT_DATA and LOG_PRINT_DELAY are associated with the console connector. 
Incorporating these options into the console datasource would be as ineffective as the URL.

It appears to me that connector options are directly populated from the connector jars, eliminating the need for additional code for these options. 
This arrangement works well for FakeSource, but it seems to be ineffective for Console. 
I will investigate why this is the case for Console.

Reply:
According to our initial idea, we will split a seatunnel connector options into two parts, one for datasource and one for connector. Finally, when generating the config, we can merge the options of these two parts. For example, for JDBC, we will put jdbc url, username, password in the datasource

https://github.com/apache/seatunnel-web/blob/main/seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-mysql/src/main/java/org/apache/seatunnel/datasource/plugin/mysql/jdbc/MysqlDataSourceConfig.java#L44

because they belong to datasource information, and put the others in the connector because different tables may have different query parameters.

So when we define an option in the datasource, the option with the same name will not be displayed when creating the connector.

This is the logic.
https://github.com/apache/seatunnel-web/blob/main/seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/thirdparty/datasource/AbstractDataSourceConfigSwitcher.java#L172

Reply:
Got it, thanks for explaining. I will add LOG_PRINT_DATA and LOG_PRINT_DELAY in console datasource.

Reply:
Added log.print.data and log.print.delay.ms options into console data source and removed URL option

![image](https://github.com/user-attachments/assets/fd622419-9fca-41a6-a94a-af18c5e83ffe)